SmbusI801.p

```diff
@@ -705,103 +726,137 @@ DEFINE_IOCTL(ioctl_smbus_xfer) {
     new read_write = in[1];
     new command = in[2];
     new hstcmd = in[3];
-    
+
+    new pci_cmd_modified;
+
+    //PCI CMD IO not enabled
+    if (false == (pci_cmd_original & PCICMD_IOBIT))
```

Comment:
I think pci_cmd_original should be fetched here instead (and probably be a local). I think you can only guarantee no others are touching this stuff while `"\BaseNamedObjects\Access_SMBUS.HTP.Method"` is held, which (according to the warning) is only while we're in this function, and not for the whole lifetime of the PawnIO module.